### PR TITLE
Mentee Add  Course

### PR DESCRIPTION
**Pull request by czarjulius**
A Mentee can add a course

### CODE REVIEW

File: app/models/mentee.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Mentee < ApplicationRecord
+  belongs_to :user
+end
```
Comment (dougbradbury) [inline app/models/mentee.rb:3]: I am confused by what a mentee is.  Is the user the mentee or the mentor?  You could name the associate for its meaning, not just the table referenced.  Check out the :foriegn_key option when defining the relationship. I am just having a hard time understanding the relationship.

File: app/controllers/mentees_controller.rb [hunk 2]
```diff
@@ -8,4 +9,13 @@ def index
       render json: { mentees: mentees }, status: :ok
     end
   end
+
+  def add_course
+    current_mentee = Mentee.find(params[:id])
+    payload = JSON.parse(request.body.read)
+    current_mentee.update!(course_id: payload['course_id'])
```
Comment (naush) [inline app/controllers/mentees_controller.rb]: Could you just access `course_id` from `params` without parsing the request payload?

File: spec/requests/mentees_request_spec.rb [hunk 3]
```diff
@@ -24,5 +24,15 @@
         expect(mentees["message"]).to eq("No mentee recorded")
       end
     end
+    context "add course" do
+      it "populate the mentee table with a course ID" do
```
Comment (naush) [inline spec/requests/mentees_request_spec.rb]: Table and ID are all implementation details. Can you think of a better test description that describes the behavior here?

**Review by chokonaira — COMMENTED** [review]
Good Job Julius, however, I just tested this endpoint locally and it seems not to be a protected route, can you check to see that only an authenticated user can make a request to this route.

**Review by Opio-Emmanuel-Omona — APPROVED** [review]
LGTM

**Review by chokonaira — APPROVED** [review]
LGTM

**Pull request merged**